src/dispatch/plugins/dispatch_core/config.py

```diff
@@ -1,3 +1,20 @@
+import logging
+
 from starlette.config import Config
 
+
+log = logging.getLogger(__name__)
+
+
 config = Config(".env")
+
+
+DISPATCH_JWT_AUDIENCE = config("DISPATCH_JWT_AUDIENCE", default=None)
+DISPATCH_JWT_EMAIL_OVERRIDE = config("DISPATCH_JWT_EMAIL_OVERRIDE", default=None)
```

Comment:
Can we document these envvars in https://hawkins.gitbook.io/dispatch/configuration/app?

Reply:
Updated